### ISSUE TITLE
Filenames should be unique

### ISSUE THREAD

**Issue by mgifford** [GovTechSG/purple-hats#19]
The reports should have URL & date hard coded into the names. So say 18f.gov-25012020-report.html & 18f.gov-25012020-compiledResult.json

This would make it easier to track.

**Comment by brianteeman** [comment]
As each scan is in its own named folder with stename and date I assume that this request has been addressed elsewhere since this issue was raised

**Comment by mgifford** [comment]
I think the package has evolved a lot since 2021 @brianteeman 

I also don't think that the maintainers engage much with the issue queue unfortunately. 

Good seeing you here!

**Comment by brianteeman** [comment]
@mgifford came back to visit it and saw a lot changes. Not sure about the privacy changes and tracking on reports though so might have to fork it

**Comment by mgifford** [comment]
Let me know what you do with it. I forked it a while back [https://github.com/civicactions/purple-hats](https://github.com/civicactions/purple-hats)   
  
Didn't have enough JS to maintain it. I did have the ability to create something that created better reports though. Least in my use case.

**Comment by brianteeman** [comment]
yes I remember your fork - but looking here it has been developed much more than when I last looked at yours
